tracing/src/level_filters.rs

```diff
@@ -95,6 +95,14 @@ impl PartialOrd<LevelFilter> for Level {
     }
 }
 
+impl LevelFilter {
```

Comment:
Rather than putting this in its own impl block, can we just put it in the impl block on line 53 (https://github.com/tokio-rs/tracing/blob/b0baa29e5cfff28d79a3068ef0e744a62f80c786/tracing/src/level_filters.rs#L53-L78)? Thanks!